src/PackingList/PackingList.js

```diff
@@ -57,7 +57,7 @@ const PackingList = ({ packingList, deleteItem, history, userInfo }) => {
           </section>
           <Link
             className='save-list-button'
-            onClick={saveNewPackingList} 
+            onClick={() => submitNewPackingList()} 
```

Comment:
I don't think we need this callback function? I think we can just pass the function definition?